instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/patches/statement.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Mysql2
+      module Patches
+        # Module to prepend to Mysql2::Client for instrumentation
+        module Statement
+          def execute(*args, **kwargs)
+            tracer.in_span(
+              'execute',
+              attributes: _otel_execute_attributes(args, kwargs),
+              kind: :client
+            ) do
+              super
+            end
+          end
+
+          private
+
+          def _otel_execute_attributes(args, kwargs)
+            if config[:db_statement] == :include
+              {'args' => args.to_s, 'kwargs' => kwargs.to_s}
```

Comment:
I do agree that we should amend all attributes without using explicit mappings for semantic conventions especially if it opens up a risk to exposing sensitive data. 

E.g. auth token or password would be exposed through telemetry collection, with a safeguards to remove them. 

I do realize we allow this to happen today with our existing `include` statement but I am leaning more and more each day to deleting that option all together and only allowing `sanitized`, `obfuscate`, or `omit` in the future.

Reply:
Does `sanitized` mean specifically removing the sensitive data (e.g. token, password, ssn, health_id, etc.), but not other binding parameters such as address?

I believe most password stored in table should be hashed.